docusaurus.config.js

```diff
@@ -41,7 +41,9 @@ const config = {
           sidebarPath: require.resolve("./sidebars.js"),
         },
         blog: {
-          showReadingTime: true,
```

Comment:
Don't we need this for the duration display??

Reply:
It's default true